docs/user-guide/src/bmo/introduction.md

```diff
@@ -59,23 +113,16 @@ status:
       type: HDD
 ```
 
-It would help to use an example to describe what BMO does. There are two operations of interest, getting hardware details of the server and  booting the server with a given image, including user supplied cloud-init data. The BareMetalHost resource contains address and authentication information towards a server.
+## Integration with the cluster API
 
-BMO communicates this information to Ironic and gets hardware details (a.k.a. inspection data), such as CPU and disk, of the server in return. This information is added to the BareMetalHost resource status. In order to get such server related information, the server is booted with service ramdisk.
-If there are hardware related changes, the BareMetalHost is updated accordingly.
+[CAPM3](../capm3/introduction) is the Metal3 component that is responsible for
+integration between Cluster API resources and BareMetalHosts. When using Metal3
+with CAPM3, you will enroll BareMetalHosts as described above first, then use
```

Comment:
```suggestion
with CAPM3, you will enroll BMHs as described above first, then use
```

Reply:
I would much prefer to keep the full name in these docs. It is hard as it is to learn a new project without having to first understand all acronyms. It is also common to do so. See for example how [cert-manager writes out CertificateRequest and CertificateSigningRequest](https://cert-manager.io/docs/usage/kube-csr/) and how [CAPI writes out all resources (e.g. MachineDeployment and KubeadmControlPlane)](https://cluster-api.sigs.k8s.io/tasks/updating-machine-templates).

Reply:
Yeah I agree, I commented for consistency since we use the acronym in the beginning of the document, perhaps we should just removed acronyms here and use BareMetelHost consistently.

Reply:
Or we could use the full form with acronym in parenthesis for the first instance (on each page if we wanted to enforce this policy), and then continue with the abbreviation later. It's only overwhelming if the resulting page has too many abbreviations, but I guess BMH and BMO etc. should be alright.